plugins/inventory/gcp_compute.py

```diff
@@ -48,6 +48,11 @@
               'public_ip', 'private_ip', 'name' or 'labels.vm_name'.
           default: ['public_ip', 'private_ip', 'name']
           type: list
+        base_domain:
```

Comment:
for the argument name: can it just be called "name_suffix"? It's going to be appended to the name of all hosts, so it' s not specific to domains.